sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BatchExecUtils.java

```diff
@@ -52,6 +52,15 @@ public static String getActivityId(Map<String, String> responseHeaders) {
         return null;
     }
 
+    public static int getResponseLength(Map<String, String> responseHeaders) {
+        int responseLength = 0;
+        if (responseHeaders != null) {
+            responseLength = Integer.parseInt(responseHeaders.get(HttpConstants.HttpHeaders.CONTENT_LENGTH));
```

Comment:
Verified right that is coming? Otherwise as discussed, we can keep length in TransactionalBatchResponse in a package private variable.

Reply:
Yes verified it.